testing/bird/APKBUILD

```diff
@@ -1,8 +1,9 @@
 # Contributor: Francesco Zanini <francesco@zanini.me>
+# Contributor: Katie Holly <holly@fuslvz.ws>
 # Maintainer: Francesco Zanini <francesco@zanini.me>
-pkgname=bird
-pkgver=1.6.0
-pkgrel=2
+pkgname="bird"
```

Comment:
This extra quoting isn't really necessary and not in line with the rest of the APKBUILDs. I'd prefer it without, personally.